LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016 Rodrigo Moraes. All rights reserved.
```

Comment:
Might be worth changing this to "The Gorilla Authors". @moraes is probably best to comment here though!